src/scores.jl

```diff
@@ -217,15 +218,22 @@ function score_partition_from_proposal(graph::BaseGraph,
     Δ_districts = [proposal.D₁, proposal.D₂]
     score_values["dists"] = Δ_districts
     for s in scores
+        value = nothing # placeholder for output of score
+        empty_return = false # whether the score returned nothing
         if s isa PlanScore
-            score_values[s.name] = eval_score_on_partition(graph, partition, s)
+            value = eval_score_on_partition(graph, partition, s)
+            empty_return = value == nothing
         elseif s isa CompositeScore
             # ensure that district-level scores in the CompositeScore are only
             # evaluated on changed districts
-            score_values[s.name] = score_partition_from_proposal(graph, partition, proposal, s.scores)
-            delete!(score_values[s.name], "dists") # remove redundant dists key
+            value = score_partition_from_proposal(graph, partition, proposal, s.scores)
+            delete!(value, "dists") # remove redundant dists key
         else # efficiently calculate & store scores only on changed districts
-            score_values[s.name] = eval_score_on_districts(graph, partition, s, Δ_districts)
+            value = eval_score_on_districts(graph, partition, s, Δ_districts)
+            empty_return = value[1] == nothing || value[2] == nothing
```

Comment:
is this the check to ensure everything in the array is `nothing`? It's only checking the first two values

Reply:
Because we know `Δ_districts` is literally defined to be `[proposal.D₁, proposal.D₂]`, `eval_score_on_districts(graph, partition, s, Δ_districts)` returns an array of length 2. A lot of our codebase is written explicitly assuming that `dists` is length 2. That being said, I changed the check to use `any` for posterity, since in the future `dists` is likely to change to accommodate >2 changed districts!

Reply:
sounds good! hadn't realized it was on `\del_districts` thank you!